crates/ruff_linter/src/rules/flake8_bandit/snapshots/ruff_linter__rules__flake8_bandit__tests__S104_S104.py.snap

```diff
@@ -15,21 +16,30 @@ S104.py:10:1: S104 Possible binding to all interfaces
  9 | "0.0.0.0"
 10 | '0.0.0.0'
    | ^^^^^^^^^ S104
+11 | f"0.0.0.0"
    |
 
-S104.py:14:6: S104 Possible binding to all interfaces
+S104.py:11:3: S104 Possible binding to all interfaces
    |
-13 | # Error
-14 | func("0.0.0.0")
+ 9 | "0.0.0.0"
+10 | '0.0.0.0'
+11 | f"0.0.0.0"
+   |   ^^^^^^^ S104
+   |
+
+S104.py:15:6: S104 Possible binding to all interfaces
```

Comment:
I think the range should include the quotes and prefix as is the case for normal strings:

```
11 | f"0.0.0.0"
   | ^^^^^^^^^^ S104
```

Same issue exists for other rules as well wherever the `FStringElement::Literal` is being used.

Reply:
that behaviour doesn't change with this pr (i'm just adding tests since i briefly regressed).

Reply:
Oh sorry, I didn't see that. Thanks for clarifying :)